addons/website_sale/views/templates.xml

```diff
@@ -1381,102 +1369,90 @@
-                                </span>
-                                <span t-else=""
-                                    t-field="line.product_id.image_128"
-                                    t-options="{'widget': 'image', 'qweb_img_responsive': False, 'class': 'rounded o_image_64_max'}"
-                                />
-                            </td>
-                            <td class='td-product_name'>
-                                <div>
-                                    <t t-call="website_sale.cart_line_product_link">
-                                        <strong t-field="line.name_short" />
-                                    </t>
-                                </div>
-                                <t t-call="website_sale.cart_line_description_following_lines">
-                                    <t t-set="div_class" t-value="'d-none d-md-block'"/>
-                                </t>
-                            </td>
-                        </t>
-                        <td class="text-center td-qty">
-                            <div class="css_quantity input-group mx-auto justify-content-center">
-                                <t t-if="not line._is_not_sellable_line()">
-                                    <t t-if="show_qty">
-                                        <a t-attf-href="#" class="btn btn-link js_add_cart_json d-none d-md-inline-block" aria-label="Remove one" title="Remove one">
-                                            <i class="fa fa-minus"></i>
-                                        </a>
-                                        <input type="text" class="js_quantity form-control quantity" t-att-data-line-id="line.id" t-att-data-product-id="line.product_id.id" t-att-value="int(line.product_uom_qty) == line.product_uom_qty and int(line.product_uom_qty) or line.product_uom_qty" />
-                                        <t t-if="line._get_shop_warning(clear=False)">
-                                            <a t-attf-href="#" class="btn btn-link">
-                                            <i class='fa fa-warning text-warning' t-att-title="line._get_shop_warning()" role="img" aria-label="Warning"/>
-                                            </a>
-                                        </t>
-                                        <a t-else='' t-attf-href="#" class="btn btn-link float_left js_add_cart_json d-none d-md-inline-block" aria-label="Add one" title="Add one">
-                                            <i class="fa fa-plus"></i>
+            <t t-foreach="website_sale_order.website_order_line" t-as="line">
+                <tr t-att-class="'optional_product info' if line.linked_line_id else None">
+                    <t t-if="not line.product_id" rowspan="2" class="td-img ps-0">
+                        <td rowspan="2" t-attf-class="td-img w-0 px-0 {{ 'border-bottom pb-4' if not line_last else '' }}"/>
+                        <td/>
```

Comment:
Let's double check if the second `<td/>` is necessary.
Eventually you may opt to use `colspan` in the first one instead 😉